codec-http2/src/test/java/io/netty/handler/codec/http2/Http2ExceptionTest.java

```diff
@@ -0,0 +1,35 @@
+package io.netty.handler.codec.http2;
+
+import io.netty.handler.codec.DecoderException;
+import io.netty.handler.codec.http.TooLongHttpLineException;
+import org.junit.jupiter.api.Test;
+
+import static io.netty.handler.codec.http2.Http2Error.*;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
please not use wildcard imports

Reply:
Apologies, I thought I had pulled in the style config from https://netty.io/wiki/setting-up-development-environment.html but may have misconfigured